service-api/app/src/App/src/Service/ViewerCodes/ViewerCodeService.php

```diff
@@ -116,17 +116,16 @@ private function populateCodeStatuses(array $codes): array
     {
         $viewerCodesAndStatuses = $this->viewerCodeActivityRepository->getStatusesForViewerCodes($codes);
 
-        // Get the actor id for the respective viewer code by UserLpaActor
+        // Get the actor id for the respective viewer code from either CreatedBy or using UserLpaActor
         foreach ($viewerCodesAndStatuses as $key => $viewerCode) {
             if (empty($viewerCode['UserLpaActor'])) {
                 $viewerCodesAndStatuses[$key]['ActorId'] = $viewerCode['CreatedBy'];
-                continue;
             }
-
-            $codeOwner = $this->getCodeOwner($viewerCode['UserLpaActor']);
-
-            if ($codeOwner !== null) {
-                $viewerCodesAndStatuses[$key]['ActorId'] = $codeOwner['ActorId'];
+            else {
+                $codeOwner = $this->getCodeOwner($viewerCode['UserLpaActor']);
+                if ($codeOwner !== NULL) {
```

Comment:
```suggestion
                if ($codeOwner !== null) {
```
I know they're equivalent but the codebase is all lowercase.